feat: DLT-2307 dialpad rebrand 2025

braddialpad opened the pull request.

josedialpad commented:
Wow this looks awesome!

braddialpad commented on docs/guides/getting-started/index.md:
(Node Package Manager)

braddialpad commented on docs/guides/getting-started/index.md:
It should be v16.x.x now. Don't need the exact minor/fix versions since they change so often.

braddialpad commented on docs/utilities/index.md:
```diff
@@ -0,0 +1,194 @@
+
+Each utility is a small, <a href="https://css-tricks.com/lets-define-exactly-atomic-css/" target="_blank">atomic style</a> declaration that, when chained together, should mitigate most situations in which custom CSS must be written. Just write these classes right in your mark-up and you're all set!
+
+Most utilities comes with responsive options, enabling alterations at targeted screen size ranges. For example, “small padding on smaller screens, larger padding on bigger screens.”
```
"come with"

braddialpad commented on docs/utilities/index.md:
```diff
@@ -0,0 +1,194 @@
+
+### 8. And change the border color
+
+Since the border color of `d-divide-x` inherits the color of the parent's foreground (implicitly `currentColor`), let's soften it with `d-divide-black-400`, from the list of [devide color classes](/utilities/borders/divide-color.html).
```
[divide color classes]

braddialpad commented on docs/utilities/index.md:
````diff
@@ -0,0 +1,194 @@
+<div class="d-bt d-btw4 d-bc-purple-400">Box</div>
+```
+
+## Tutorial
````
Super nice tutorial here.

braddialpad commented on docs/guides/getting-started/index.md:
````diff
@@ -1,9 +1,47 @@
 ---
 
-## Utility-first
+## Adding Dialtone to your project
+
+To take advantage of Dialtone's customizations, classes, and variables in your project, you will want to install Dialtone via <a href="https://www.npmjs.com/" target="_blank">npm</a>.
+
+```bash
+npm install @dialpad/dialtone
+```
+
+Add the following line in your Less file:
+
+```less
+@import "your/path/to/dialtone.less";
+```
+
+If you only need access to Dialtone's variables and customizations to build a file and don't need the whole library exported, use this line instead in your Less file:
+
+```less
+@import (reference) "your/path/to/dialtone.less";
+```
+
+## Theming
+
+To customize your Dialtone installation, create a new Less file in the `/lib/build/less/themes/` folder. By default this will compile during build, which can be added as an additional CSS file in your page `head`.
+
+```less
+@import "your/path/to/dialtone.less";
+```
+
+If you only need access to Dialtone's variables and customizations to build a file and don't need the whole library exported, use this line instead in your Less file:
+
+```less
+@import (reference) "your/path/to/dialtone.less";
+```
````
This section might need some updating to recommend css vars instead of less vars, better examples for things like theming, but I think this is a good enough for the purposes of this PR

francisrupert replied:
Logged in [DT-811](https://dialpad.atlassian.net/browse/DT-811). No action for this PR.

francisrupert commented:
Holding off on merging. Having Elise review content for intro statement and card blurbs.

francisrupert committed:
base deca to default (dp)

francisrupert committed:
base deca dark to default dark (dp)

francisrupert committed:
semantic deca to default (dp) - foreground and link

francisrupert committed:
semantic deca to default (dp) - surfaces

francisrupert committed:
semantic deca to default (dp) - border

francisrupert committed:
semantic deca to default (dp) - action

francisrupert committed:
semantic deca to default (dp) - theme

francisrupert committed:
semantic deca to default (dp) - theme

francisrupert committed:
semantic deca to default (dp) - theme

francisrupert committed:
semantic deca to dark (dp) - border

francisrupert committed:
semantic deca to dark (dp) - theme

francisrupert committed:
semantic deca to dark (dp) - action

francisrupert committed:
semantic deca to default (dp) - badge

francisrupert committed:
semantic deca to dp - badge

francisrupert committed:
semantic deca to dp - button

francisrupert committed:
semantic deca to dp - checkbox

francisrupert committed:
semantic deca to dp - radio

francisrupert committed:
minimized deca set references

francisrupert committed:
correct gold

francisrupert committed:
critical foreground correction b/t light inverted and dark default

francisrupert committed:
remove asdf

francisrupert committed:
move deca to default (dp): inputs

francisrupert committed:
remove dark overrides, deferring to inherited default - inputs

francisrupert committed:
correct primary opaque

francisrupert committed:
primary-opaque-inverted surface

francisrupert committed:
figma styles and theme config

juliodialpad committed:
ci: DLT-2366 add branch to deploy

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
ci: DLT-2365 add branch to prerelease

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
update release branches

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.101.0-rebrand-2025-beta.1

# [9.101.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.100.0...dialtone/v9.101.0-rebrand-2025-beta.1) (2025-02-17)

### Documentation

* NO-JIRA fix ivr-node ([#630](https://github.com/dialpad/dialtone/issues/630)) ([bca81d8](https://github.com/dialpad/dialtone/commit/bca81d8ac1f0701512a35c6af704762c08d8f080))

### Features

* **Design Tokens:** NO-JIRA experimental deca token set updates again ([#632](https://github.com/dialpad/dialtone/issues/632)) ([c7ad513](https://github.com/dialpad/dialtone/commit/c7ad513f49215728dae86bf3305dd54f44df1251))

juliodialpad committed:
ci: DLT-2365 add branch to prerelease publish

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
Merge remote-tracking branch 'origin/rebrand-2025-beta' into rebrand-2025-beta

francisrupert committed:
Update README.md

.

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

# Conflicts:
#	CHANGELOG.json
#	CHANGELOG.md
#	package.json

juliodialpad committed:
add rebrand-2025-beta branch to clean exclude

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

francisrupert committed:
feat(design-tokens): DLT-2352 apply new 2025 reb colors (#636)

Co-authored-by: Brad Paugh <brad.paugh@dialpad.com>

juliodialpad committed:
feat: DLT-2329 update Archivo to Season Mix font (#648)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>
Co-authored-by: Francis Rupert <francisrupert@gmail.com>

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

# Conflicts:
#	.github/workflows/deploy.yml

jcarter-dp committed:
feat(empty-state): DLT-2361 DLT-2362 update icon color (#651)

Updated PR title to specify the change.

francisrupert committed:
docs: DLT-2354 DLT-2331 apply season to doc site and add dialtone logo (#652)

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

juliodialpad committed:
feat: DLT-2350 expand color css utility classes (#655)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

juliodialpad committed:
ci: NO-JIRA add rebrand 2025 beta to visual tests (#661)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
fix(contact-centers-row): DLT-2363 update colors on recipes (#663)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
docs: DLT-2391 color utilities documentation (#659)

francisrupert committed:
feat(design-tokens): NO-JIRA updated purples

francisrupert committed:
feat(design-tokens): NO-JIRA updated purples dark mode library connection

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.105.0-rebrand-2025-beta.1

# [9.105.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.104.0...dialtone/v9.105.0-rebrand-2025-beta.1) (2025-03-11)

### Bug Fixes

* **Contact Centers Row:** DLT-2363 update colors on recipes ([#663](https://github.com/dialpad/dialtone/issues/663)) ([aac54e6](https://github.com/dialpad/dialtone/commit/aac54e61185938ea5b73fcaae31ccfca4c5b20b4))

### Documentation

* DLT-2354 DLT-2331 apply season to doc site and add dialtone logo ([#652](https://github.com/dialpad/dialtone/issues/652)) ([0bd67c5](https://github.com/dialpad/dialtone/commit/0bd67c52cf8b01085b87041c329969d1f901f6ae))
* DLT-2391 color utilities documentation ([#659](https://github.com/dialpad/dialtone/issues/659)) ([4d07c12](https://github.com/dialpad/dialtone/commit/4d07c126efd6d29b54f5b0ec599127cc39e84d6c))
* NO-JIRA automate utility classes extraction ([#654](https://github.com/dialpad/dialtone/issues/654)) ([7a53512](https://github.com/dialpad/dialtone/commit/7a53512046d6403958aafda6aa93caab6d2bc3b8))

### Features

* **Design Tokens:** DLT-2352 apply new 2025 reb colors ([#636](https://github.com/dialpad/dialtone/issues/636)) ([40d8f96](https://github.com/dialpad/dialtone/commit/40d8f9629ca770cad4a9f9cfd18002824c3302d1))
* **Design Tokens:** NO-JIRA updated purples ([099fa4b](https://github.com/dialpad/dialtone/commit/099fa4b2ed2050688142711f607558b608fe31c0))
* **Design Tokens:** NO-JIRA updated purples dark mode library connection ([a5aa5ab](https://github.com/dialpad/dialtone/commit/a5aa5ab02177c5f6c2a8cfdba4bb2e875278c7b8))
* DLT-2329 update Archivo to Season Mix font ([#648](https://github.com/dialpad/dialtone/issues/648)) ([ca8fdcf](https://github.com/dialpad/dialtone/commit/ca8fdcf6c3064d402a87be050b00fdbee92d6f88))
* DLT-2350 expand color css utility classes ([#655](https://github.com/dialpad/dialtone/issues/655)) ([1b271f3](https://github.com/dialpad/dialtone/commit/1b271f36eafbd798cdaa8c38f40202e11719909f))
* **Empty State:** DLT-2361 DLT-2362 update icon color ([#651](https://github.com/dialpad/dialtone/issues/651)) ([0be7414](https://github.com/dialpad/dialtone/commit/0be7414bd2d2ef32d2d491e638ebdd492971746c))

francisrupert committed:
feat(design-tokens): NO-JIRA update critical foreground and border

fede-dp committed:
docs: modify the brand page, remove the word-mark variant, update the logo (#668)

Co-authored-by: Francis Rupert <francis.rupert@dialpad.com>

francisrupert committed:
docs(css-utilities): NO-JIRA update css utilities demo to emphasize semantic (#665)

francisrupert committed:
feat(design-tokens): NO-JIRA updated dark muted foreground color

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.105.0-rebrand-2025-beta.2

# [9.105.0-rebrand-2025-beta.2](https://github.com/dialpad/dialtone/compare/dialtone/v9.105.0-rebrand-2025-beta.1...dialtone/v9.105.0-rebrand-2025-beta.2) (2025-03-12)

### Documentation

* **Css Utilities:** NO-JIRA update css utilities demo to emphasize semantic ([#665](https://github.com/dialpad/dialtone/issues/665)) ([520eab4](https://github.com/dialpad/dialtone/commit/520eab42a48b6d641f156cb9059ac76416dcdd24))
* modify the brand page, remove the word-mark variant, update the logo ([#668](https://github.com/dialpad/dialtone/issues/668)) ([08370f7](https://github.com/dialpad/dialtone/commit/08370f7aa45a95e1ba2e0cedb253a98215be5b2c))

### Features

* **Design Tokens:** NO-JIRA update critical foreground and border ([ebd2885](https://github.com/dialpad/dialtone/commit/ebd288508696f4c3aa3fd720b1122d86662f6531))
* **Design Tokens:** NO-JIRA updated dark muted foreground color ([f5146cc](https://github.com/dialpad/dialtone/commit/f5146cca5eb3fbff4d3cad6faf381a6eb2510c99))

juliodialpad committed:
feat(linters): DLT-2395 base color linter rules (#667)

juliodialpad committed:
Merge remote-tracking branch 'origin/staging' into rebrand-2025-beta

# Conflicts:
#	CHANGELOG.json
#	CHANGELOG.md
#	package.json

francisrupert committed:
feat(design-tokens): NO-JIRA unhack tmo theme (#669)

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.106.0-rebrand-2025-beta.1

# [9.106.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.105.0...dialtone/v9.106.0-rebrand-2025-beta.1) (2025-03-13)

### Bug Fixes

* **Contact Centers Row:** DLT-2363 update colors on recipes ([#663](https://github.com/dialpad/dialtone/issues/663)) ([aac54e6](https://github.com/dialpad/dialtone/commit/aac54e61185938ea5b73fcaae31ccfca4c5b20b4))

### Documentation

* **Css Utilities:** NO-JIRA update css utilities demo to emphasize semantic ([#665](https://github.com/dialpad/dialtone/issues/665)) ([520eab4](https://github.com/dialpad/dialtone/commit/520eab42a48b6d641f156cb9059ac76416dcdd24))
* DLT-2354 DLT-2331 apply season to doc site and add dialtone logo ([#652](https://github.com/dialpad/dialtone/issues/652)) ([0bd67c5](https://github.com/dialpad/dialtone/commit/0bd67c52cf8b01085b87041c329969d1f901f6ae))
* DLT-2391 color utilities documentation ([#659](https://github.com/dialpad/dialtone/issues/659)) ([4d07c12](https://github.com/dialpad/dialtone/commit/4d07c126efd6d29b54f5b0ec599127cc39e84d6c))
* modify the brand page, remove the word-mark variant, update the logo ([#668](https://github.com/dialpad/dialtone/issues/668)) ([08370f7](https://github.com/dialpad/dialtone/commit/08370f7aa45a95e1ba2e0cedb253a98215be5b2c))

### Features

* **Design Tokens:** DLT-2352 apply new 2025 reb colors ([#636](https://github.com/dialpad/dialtone/issues/636)) ([40d8f96](https://github.com/dialpad/dialtone/commit/40d8f9629ca770cad4a9f9cfd18002824c3302d1))
* **Design Tokens:** NO-JIRA unhack tmo theme ([#669](https://github.com/dialpad/dialtone/issues/669)) ([daa0801](https://github.com/dialpad/dialtone/commit/daa0801aac82342129f1cfd13849c62063064422))
* **Design Tokens:** NO-JIRA update critical foreground and border ([ebd2885](https://github.com/dialpad/dialtone/commit/ebd288508696f4c3aa3fd720b1122d86662f6531))
* **Design Tokens:** NO-JIRA updated dark muted foreground color ([f5146cc](https://github.com/dialpad/dialtone/commit/f5146cca5eb3fbff4d3cad6faf381a6eb2510c99))
* **Design Tokens:** NO-JIRA updated purples ([099fa4b](https://github.com/dialpad/dialtone/commit/099fa4b2ed2050688142711f607558b608fe31c0))
* **Design Tokens:** NO-JIRA updated purples dark mode library connection ([a5aa5ab](https://github.com/dialpad/dialtone/commit/a5aa5ab02177c5f6c2a8cfdba4bb2e875278c7b8))
* DLT-2329 update Archivo to Season Mix font ([#648](https://github.com/dialpad/dialtone/issues/648)) ([ca8fdcf](https://github.com/dialpad/dialtone/commit/ca8fdcf6c3064d402a87be050b00fdbee92d6f88))
* DLT-2350 expand color css utility classes ([#655](https://github.com/dialpad/dialtone/issues/655)) ([1b271f3](https://github.com/dialpad/dialtone/commit/1b271f36eafbd798cdaa8c38f40202e11719909f))
* **Empty State:** DLT-2361 DLT-2362 update icon color ([#651](https://github.com/dialpad/dialtone/issues/651)) ([0be7414](https://github.com/dialpad/dialtone/commit/0be7414bd2d2ef32d2d491e638ebdd492971746c))
* **Linters:** DLT-2395 base color linter rules ([#667](https://github.com/dialpad/dialtone/issues/667)) ([a23e3c6](https://github.com/dialpad/dialtone/commit/a23e3c6b4229ecd70b1ce9766b30a8f5ba42e2ad))

fede-dp committed:
feat(icon): DLT-2410 new dialpad logo icon and update gradients (#670)

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.106.0-rebrand-2025-beta.2

# [9.106.0-rebrand-2025-beta.2](https://github.com/dialpad/dialtone/compare/dialtone/v9.106.0-rebrand-2025-beta.1...dialtone/v9.106.0-rebrand-2025-beta.2) (2025-03-14)

### Features

* **Icon:** DLT-2410 new dialpad logo icon and update gradients ([#670](https://github.com/dialpad/dialtone/issues/670)) ([c29ee5a](https://github.com/dialpad/dialtone/commit/c29ee5aa2898bab28b4494fe70050fedaff1fb86))

juliodialpad committed:
fix: NO-JIRA icons build, naming conflict and typo (#671)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
add emojis, icons and tokens to rebrand release process

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
Merge remote-tracking branch 'origin/rebrand-2025-beta' into rebrand-2025-beta

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.1

# [4.35.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.34.1...dialtone-icons/v4.35.0-rebrand-2025-beta.1) (2025-03-14)

### Bug Fixes

* NO-JIRA icons build, naming conflict and typo ([#671](https://github.com/dialpad/dialtone/issues/671)) ([de83b22](https://github.com/dialpad/dialtone/commit/de83b22354ab59fa52af91fbbdf8f8060ac5b88c))

### Documentation

* modify the brand page, remove the word-mark variant, update the logo ([#668](https://github.com/dialpad/dialtone/issues/668)) ([08370f7](https://github.com/dialpad/dialtone/commit/08370f7aa45a95e1ba2e0cedb253a98215be5b2c))

### Features

* **Icon:** DLT-2410 new dialpad logo icon and update gradients ([#670](https://github.com/dialpad/dialtone/issues/670)) ([c29ee5a](https://github.com/dialpad/dialtone/commit/c29ee5aa2898bab28b4494fe70050fedaff1fb86))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.1 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-tokens/v1.42.0-rebrand-2025-beta.1

# [1.42.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone-tokens/v1.41.1...dialtone-tokens/v1.42.0-rebrand-2025-beta.1) (2025-03-14)

### Features

* **Design Tokens:** DLT-2352 apply new 2025 reb colors ([#636](https://github.com/dialpad/dialtone/issues/636)) ([40d8f96](https://github.com/dialpad/dialtone/commit/40d8f9629ca770cad4a9f9cfd18002824c3302d1))
* **Design Tokens:** NO-JIRA unhack tmo theme ([#669](https://github.com/dialpad/dialtone/issues/669)) ([daa0801](https://github.com/dialpad/dialtone/commit/daa0801aac82342129f1cfd13849c62063064422))
* **Design Tokens:** NO-JIRA update critical foreground and border ([ebd2885](https://github.com/dialpad/dialtone/commit/ebd288508696f4c3aa3fd720b1122d86662f6531))
* **Design Tokens:** NO-JIRA updated dark muted foreground color ([f5146cc](https://github.com/dialpad/dialtone/commit/f5146cca5eb3fbff4d3cad6faf381a6eb2510c99))
* **Design Tokens:** NO-JIRA updated purples ([099fa4b](https://github.com/dialpad/dialtone/commit/099fa4b2ed2050688142711f607558b608fe31c0))
* **Design Tokens:** NO-JIRA updated purples dark mode library connection ([a5aa5ab](https://github.com/dialpad/dialtone/commit/a5aa5ab02177c5f6c2a8cfdba4bb2e875278c7b8))
* DLT-2329 update Archivo to Season Mix font ([#648](https://github.com/dialpad/dialtone/issues/648)) ([ca8fdcf](https://github.com/dialpad/dialtone/commit/ca8fdcf6c3064d402a87be050b00fdbee92d6f88))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-tokens/v1.42.0-rebrand-2025-beta.1 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.106.0-rebrand-2025-beta.3

# [9.106.0-rebrand-2025-beta.3](https://github.com/dialpad/dialtone/compare/dialtone/v9.106.0-rebrand-2025-beta.2...dialtone/v9.106.0-rebrand-2025-beta.3) (2025-03-14)

### Bug Fixes

* NO-JIRA icons build, naming conflict and typo ([#671](https://github.com/dialpad/dialtone/issues/671)) ([de83b22](https://github.com/dialpad/dialtone/commit/de83b22354ab59fa52af91fbbdf8f8060ac5b88c))

juliodialpad committed:
fix: NO-JIRA bring dialpad illustration assets back

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.2

# [4.35.0-rebrand-2025-beta.2](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.1...dialtone-icons/v4.35.0-rebrand-2025-beta.2) (2025-03-14)

### Bug Fixes

* NO-JIRA bring dialpad illustration assets back ([e3dc23e](https://github.com/dialpad/dialtone/commit/e3dc23eb1418b9901a6a6eabd002c2e760a7a4e3))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.2 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.106.0-rebrand-2025-beta.4

# [9.106.0-rebrand-2025-beta.4](https://github.com/dialpad/dialtone/compare/dialtone/v9.106.0-rebrand-2025-beta.3...dialtone/v9.106.0-rebrand-2025-beta.4) (2025-03-14)

### Bug Fixes

* NO-JIRA bring dialpad illustration assets back ([e3dc23e](https://github.com/dialpad/dialtone/commit/e3dc23eb1418b9901a6a6eabd002c2e760a7a4e3))

juliodialpad committed:
fix: NO-JIRA icons-illustrations naming conflicts

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
Merge remote-tracking branch 'origin/rebrand-2025-beta' into rebrand-2025-beta

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.3

# [4.35.0-rebrand-2025-beta.3](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.2...dialtone-icons/v4.35.0-rebrand-2025-beta.3) (2025-03-14)

### Bug Fixes

* NO-JIRA icons-illustrations naming conflicts ([29f1f13](https://github.com/dialpad/dialtone/commit/29f1f13f1cd55af95327f7900b88d018f327ab32))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.3 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.106.0-rebrand-2025-beta.5

# [9.106.0-rebrand-2025-beta.5](https://github.com/dialpad/dialtone/compare/dialtone/v9.106.0-rebrand-2025-beta.4...dialtone/v9.106.0-rebrand-2025-beta.5) (2025-03-15)

### Bug Fixes

* NO-JIRA icons-illustrations naming conflicts ([29f1f13](https://github.com/dialpad/dialtone/commit/29f1f13f1cd55af95327f7900b88d018f327ab32))

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

# Conflicts:
#	CHANGELOG.json
#	CHANGELOG.md
#	package.json
#	pnpm-lock.yaml

juliodialpad committed:
update pnpm lock

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

fede-dp committed:
feat(icon): DLT-2426 add dialpad meetings icon (#673)

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.4

# [4.35.0-rebrand-2025-beta.4](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.3...dialtone-icons/v4.35.0-rebrand-2025-beta.4) (2025-03-19)

### Features

* **Icon:** DLT-2426 add dialpad meetings icon ([#673](https://github.com/dialpad/dialtone/issues/673)) ([96ba523](https://github.com/dialpad/dialtone/commit/96ba52320ff65ff281c4f35e744ed35c1c94fa58))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.4 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.106.0-rebrand-2025-beta.6

# [9.106.0-rebrand-2025-beta.6](https://github.com/dialpad/dialtone/compare/dialtone/v9.106.0-rebrand-2025-beta.5...dialtone/v9.106.0-rebrand-2025-beta.6) (2025-03-19)

### Bug Fixes

* NO-JIRA update date-fns to latest version ([#672](https://github.com/dialpad/dialtone/issues/672)) ([1ec525e](https://github.com/dialpad/dialtone/commit/1ec525ecbb8166c4a4addd2d4292b58f6ce5ad2c))

### Features

* **Icon:** DLT-2426 add dialpad meetings icon ([#673](https://github.com/dialpad/dialtone/issues/673)) ([96ba523](https://github.com/dialpad/dialtone/commit/96ba52320ff65ff281c4f35e744ed35c1c94fa58))

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

# Conflicts:
#	CHANGELOG.json
#	CHANGELOG.md
#	package.json

juliodialpad committed:
docs: DLT-2411 suppress-old-brand-icons (#680)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
ci: NO-JIRA publish rebrand-2025-beta android tokens and icons

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
refactor: DLT-2334 update design colors documentation (#679)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

juliodialpad committed:
feat(button): DLT-2408 add positive kind (#681)

francisrupert committed:
feat(design-tokens): NO-JIRA positive button border color

francisrupert committed:
feat(design-tokens): NO-JIRA positive button border color (dark)

francisrupert committed:
feat(design-tokens): NO-JIRA remap focus color

juliodialpad committed:
merge staging

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
fix action

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
fix android action

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-tokens/v1.42.0-rebrand-2025-beta.2

# [1.42.0-rebrand-2025-beta.2](https://github.com/dialpad/dialtone/compare/dialtone-tokens/v1.42.0-rebrand-2025-beta.1...dialtone-tokens/v1.42.0-rebrand-2025-beta.2) (2025-04-03)

### Bug Fixes

* **Ci:** NO-JIRA changelog release order ([#689](https://github.com/dialpad/dialtone/issues/689)) ([bd4c080](https://github.com/dialpad/dialtone/commit/bd4c0802a59be6fdafb8f661a9c12c64ef764656))

### Features

* **Design Tokens:** NO-JIRA positive button border color ([497bba8](https://github.com/dialpad/dialtone/commit/497bba84dd899e7a01dbe5a7c21741c0c4572d02))
* **Design Tokens:** NO-JIRA positive button border color (dark) ([82b8654](https://github.com/dialpad/dialtone/commit/82b8654f0a3f9712188373e371bbb32e6066f8be))
* **Design Tokens:** NO-JIRA remap focus color ([e003c8b](https://github.com/dialpad/dialtone/commit/e003c8b8e38cda3a627b48eae48c1e8e6649bfbd))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-tokens/v1.42.0-rebrand-2025-beta.2 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.5

# [4.35.0-rebrand-2025-beta.5](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.4...dialtone-icons/v4.35.0-rebrand-2025-beta.5) (2025-04-03)

### Bug Fixes

* **Ci:** NO-JIRA changelog release order ([#689](https://github.com/dialpad/dialtone/issues/689)) ([bd4c080](https://github.com/dialpad/dialtone/commit/bd4c0802a59be6fdafb8f661a9c12c64ef764656))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.5 gradle

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

francisrupert committed:
feat(design-tokens): DLT-2458 wfm extended base colors (#694)

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.6

# [4.35.0-rebrand-2025-beta.6](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.5...dialtone-icons/v4.35.0-rebrand-2025-beta.6) (2025-04-07)

### Bug Fixes

* NO-JIRA dialtone icons android build ([#691](https://github.com/dialpad/dialtone/issues/691)) ([8c10768](https://github.com/dialpad/dialtone/commit/8c10768d86c7d93707541971287c6de283d9bb85))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.6 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-tokens/v1.42.0-rebrand-2025-beta.3

# [1.42.0-rebrand-2025-beta.3](https://github.com/dialpad/dialtone/compare/dialtone-tokens/v1.42.0-rebrand-2025-beta.2...dialtone-tokens/v1.42.0-rebrand-2025-beta.3) (2025-04-07)

### Features

* **Design Tokens:** DLT-2458 wfm extended base colors ([#694](https://github.com/dialpad/dialtone/issues/694)) ([2905279](https://github.com/dialpad/dialtone/commit/2905279609c8abcb0ac2168328b912fc950d3dfc))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-tokens/v1.42.0-rebrand-2025-beta.3 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.108.0-rebrand-2025-beta.1

# [9.108.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.107.1...dialtone/v9.108.0-rebrand-2025-beta.1) (2025-04-07)

### Bug Fixes

* **Contact Centers Row:** DLT-2363 update colors on recipes ([#663](https://github.com/dialpad/dialtone/issues/663)) ([aac54e6](https://github.com/dialpad/dialtone/commit/aac54e61185938ea5b73fcaae31ccfca4c5b20b4))
* NO-JIRA $el in vue 3 ([#690](https://github.com/dialpad/dialtone/issues/690)) ([0027ba7](https://github.com/dialpad/dialtone/commit/0027ba7dbe3c394b9185797790a6408a6eae5234))
* NO-JIRA bring dialpad illustration assets back ([e3dc23e](https://github.com/dialpad/dialtone/commit/e3dc23eb1418b9901a6a6eabd002c2e760a7a4e3))
* NO-JIRA dialtone icons android build ([#691](https://github.com/dialpad/dialtone/issues/691)) ([8c10768](https://github.com/dialpad/dialtone/commit/8c10768d86c7d93707541971287c6de283d9bb85))
* NO-JIRA icons build, naming conflict and typo ([#671](https://github.com/dialpad/dialtone/issues/671)) ([de83b22](https://github.com/dialpad/dialtone/commit/de83b22354ab59fa52af91fbbdf8f8060ac5b88c))
* NO-JIRA icons-illustrations naming conflicts ([29f1f13](https://github.com/dialpad/dialtone/commit/29f1f13f1cd55af95327f7900b88d018f327ab32))
* **Tooltip:** DP-129564 add overflow-wrap for tooltip ([#687](https://github.com/dialpad/dialtone/issues/687)) ([b6d8256](https://github.com/dialpad/dialtone/commit/b6d825647b22b1c18685a895b84985d76a6ce938))

### Code Refactoring

* DLT-2334 update design colors documentation ([#679](https://github.com/dialpad/dialtone/issues/679)) ([a22a1c0](https://github.com/dialpad/dialtone/commit/a22a1c0b138f147419229c8c182c2cd4d619e66f))

### Documentation

* **Css Utilities:** NO-JIRA update css utilities demo to emphasize semantic ([#665](https://github.com/dialpad/dialtone/issues/665)) ([520eab4](https://github.com/dialpad/dialtone/commit/520eab42a48b6d641f156cb9059ac76416dcdd24))
* DLT-2354 DLT-2331 apply season to doc site and add dialtone logo ([#652](https://github.com/dialpad/dialtone/issues/652)) ([0bd67c5](https://github.com/dialpad/dialtone/commit/0bd67c52cf8b01085b87041c329969d1f901f6ae))
* DLT-2391 color utilities documentation ([#659](https://github.com/dialpad/dialtone/issues/659)) ([4d07c12](https://github.com/dialpad/dialtone/commit/4d07c126efd6d29b54f5b0ec599127cc39e84d6c))
* DLT-2411 suppress-old-brand-icons ([#680](https://github.com/dialpad/dialtone/issues/680)) ([7f6a35e](https://github.com/dialpad/dialtone/commit/7f6a35e585694e4d594ad73d36ed2ebcf1edf568))
* modify the brand page, remove the word-mark variant, update the logo ([#668](https://github.com/dialpad/dialtone/issues/668)) ([08370f7](https://github.com/dialpad/dialtone/commit/08370f7aa45a95e1ba2e0cedb253a98215be5b2c))

### Features

* **Button:** DLT-2408 add positive kind ([#681](https://github.com/dialpad/dialtone/issues/681)) ([0265764](https://github.com/dialpad/dialtone/commit/0265764a631c8a909b1dfa288d5e21881aac0d8e))
* **Design Tokens:** DLT-2352 apply new 2025 reb colors ([#636](https://github.com/dialpad/dialtone/issues/636)) ([40d8f96](https://github.com/dialpad/dialtone/commit/40d8f9629ca770cad4a9f9cfd18002824c3302d1))
* **Design Tokens:** DLT-2458 wfm extended base colors ([#694](https://github.com/dialpad/dialtone/issues/694)) ([2905279](https://github.com/dialpad/dialtone/commit/2905279609c8abcb0ac2168328b912fc950d3dfc))
* **Design Tokens:** NO-JIRA positive button border color ([497bba8](https://github.com/dialpad/dialtone/commit/497bba84dd899e7a01dbe5a7c21741c0c4572d02))
* **Design Tokens:** NO-JIRA positive button border color (dark) ([82b8654](https://github.com/dialpad/dialtone/commit/82b8654f0a3f9712188373e371bbb32e6066f8be))
* **Design Tokens:** NO-JIRA remap focus color ([e003c8b](https://github.com/dialpad/dialtone/commit/e003c8b8e38cda3a627b48eae48c1e8e6649bfbd))
* **Design Tokens:** NO-JIRA unhack tmo theme ([#669](https://github.com/dialpad/dialtone/issues/669)) ([daa0801](https://github.com/dialpad/dialtone/commit/daa0801aac82342129f1cfd13849c62063064422))
* **Design Tokens:** NO-JIRA update critical foreground and border ([ebd2885](https://github.com/dialpad/dialtone/commit/ebd288508696f4c3aa3fd720b1122d86662f6531))
* **Design Tokens:** NO-JIRA updated dark muted foreground color ([f5146cc](https://github.com/dialpad/dialtone/commit/f5146cca5eb3fbff4d3cad6faf381a6eb2510c99))
* **Design Tokens:** NO-JIRA updated purples ([099fa4b](https://github.com/dialpad/dialtone/commit/099fa4b2ed2050688142711f607558b608fe31c0))
* **Design Tokens:** NO-JIRA updated purples dark mode library connection ([a5aa5ab](https://github.com/dialpad/dialtone/commit/a5aa5ab02177c5f6c2a8cfdba4bb2e875278c7b8))
* DLT-2329 update Archivo to Season Mix font ([#648](https://github.com/dialpad/dialtone/issues/648)) ([ca8fdcf](https://github.com/dialpad/dialtone/commit/ca8fdcf6c3064d402a87be050b00fdbee92d6f88))
* DLT-2350 expand color css utility classes ([#655](https://github.com/dialpad/dialtone/issues/655)) ([1b271f3](https://github.com/dialpad/dialtone/commit/1b271f36eafbd798cdaa8c38f40202e11719909f))
* **Empty State:** DLT-2361 DLT-2362 update icon color ([#651](https://github.com/dialpad/dialtone/issues/651)) ([0be7414](https://github.com/dialpad/dialtone/commit/0be7414bd2d2ef32d2d491e638ebdd492971746c))
* **Icon:** DLT-2410 new dialpad logo icon and update gradients ([#670](https://github.com/dialpad/dialtone/issues/670)) ([c29ee5a](https://github.com/dialpad/dialtone/commit/c29ee5aa2898bab28b4494fe70050fedaff1fb86))
* **Icon:** DLT-2426 add dialpad meetings icon ([#673](https://github.com/dialpad/dialtone/issues/673)) ([96ba523](https://github.com/dialpad/dialtone/commit/96ba52320ff65ff281c4f35e744ed35c1c94fa58))
* **Linters:** DLT-2395 base color linter rules ([#667](https://github.com/dialpad/dialtone/issues/667)) ([a23e3c6](https://github.com/dialpad/dialtone/commit/a23e3c6b4229ecd70b1ce9766b30a8f5ba42e2ad))

francisrupert committed:
chore: NO-JIRA scrollable doc tables (#695)

Co-authored-by: Julio Ortega <julio.ortega@dialpad.com>

juliodialpad committed:
merge staging

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

fede-dp committed:
feat(icons): [NO-JIRA] new icons dialpad-star (#706)

fede-dp committed:
feat(illustrations): DLT-2424 add rebranded illustrations (#693)

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>
Co-authored-by: Julio Ortega <julio.ortega@dialpad.com>

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.7

# [4.35.0-rebrand-2025-beta.7](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.6...dialtone-icons/v4.35.0-rebrand-2025-beta.7) (2025-04-11)

### Features

* **Icons:** [NO-JIRA] new icons dialpad-star ([#706](https://github.com/dialpad/dialtone/issues/706)) ([f38fefd](https://github.com/dialpad/dialtone/commit/f38fefd0b4bcd9de7780cb3c48b291d9de8dc29a))
* **Illustrations:** DLT-2424 add rebranded illustrations ([#693](https://github.com/dialpad/dialtone/issues/693)) ([6644761](https://github.com/dialpad/dialtone/commit/6644761f32a3854e0f404cd7416d9a12805bee38))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.7 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.109.0-rebrand-2025-beta.1

# [9.109.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.108.0...dialtone/v9.109.0-rebrand-2025-beta.1) (2025-04-11)

### Bug Fixes

* **Contact Centers Row:** DLT-2363 update colors on recipes ([#663](https://github.com/dialpad/dialtone/issues/663)) ([aac54e6](https://github.com/dialpad/dialtone/commit/aac54e61185938ea5b73fcaae31ccfca4c5b20b4))
* NO-JIRA bring dialpad illustration assets back ([e3dc23e](https://github.com/dialpad/dialtone/commit/e3dc23eb1418b9901a6a6eabd002c2e760a7a4e3))
* NO-JIRA icons build, naming conflict and typo ([#671](https://github.com/dialpad/dialtone/issues/671)) ([de83b22](https://github.com/dialpad/dialtone/commit/de83b22354ab59fa52af91fbbdf8f8060ac5b88c))
* NO-JIRA icons-illustrations naming conflicts ([29f1f13](https://github.com/dialpad/dialtone/commit/29f1f13f1cd55af95327f7900b88d018f327ab32))

### Code Refactoring

* DLT-2334 update design colors documentation ([#679](https://github.com/dialpad/dialtone/issues/679)) ([a22a1c0](https://github.com/dialpad/dialtone/commit/a22a1c0b138f147419229c8c182c2cd4d619e66f))

### Documentation

* **Css Utilities:** NO-JIRA update css utilities demo to emphasize semantic ([#665](https://github.com/dialpad/dialtone/issues/665)) ([520eab4](https://github.com/dialpad/dialtone/commit/520eab42a48b6d641f156cb9059ac76416dcdd24))
* DLT-2354 DLT-2331 apply season to doc site and add dialtone logo ([#652](https://github.com/dialpad/dialtone/issues/652)) ([0bd67c5](https://github.com/dialpad/dialtone/commit/0bd67c52cf8b01085b87041c329969d1f901f6ae))
* DLT-2391 color utilities documentation ([#659](https://github.com/dialpad/dialtone/issues/659)) ([4d07c12](https://github.com/dialpad/dialtone/commit/4d07c126efd6d29b54f5b0ec599127cc39e84d6c))
* DLT-2411 suppress-old-brand-icons ([#680](https://github.com/dialpad/dialtone/issues/680)) ([7f6a35e](https://github.com/dialpad/dialtone/commit/7f6a35e585694e4d594ad73d36ed2ebcf1edf568))
* modify the brand page, remove the word-mark variant, update the logo ([#668](https://github.com/dialpad/dialtone/issues/668)) ([08370f7](https://github.com/dialpad/dialtone/commit/08370f7aa45a95e1ba2e0cedb253a98215be5b2c))

### Features

* **Button:** DLT-2408 add positive kind ([#681](https://github.com/dialpad/dialtone/issues/681)) ([0265764](https://github.com/dialpad/dialtone/commit/0265764a631c8a909b1dfa288d5e21881aac0d8e))
* **Design Tokens:** DLT-2352 apply new 2025 reb colors ([#636](https://github.com/dialpad/dialtone/issues/636)) ([40d8f96](https://github.com/dialpad/dialtone/commit/40d8f9629ca770cad4a9f9cfd18002824c3302d1))
* **Design Tokens:** DLT-2458 wfm extended base colors ([#694](https://github.com/dialpad/dialtone/issues/694)) ([2905279](https://github.com/dialpad/dialtone/commit/2905279609c8abcb0ac2168328b912fc950d3dfc))
* **Design Tokens:** NO-JIRA positive button border color ([497bba8](https://github.com/dialpad/dialtone/commit/497bba84dd899e7a01dbe5a7c21741c0c4572d02))
* **Design Tokens:** NO-JIRA positive button border color (dark) ([82b8654](https://github.com/dialpad/dialtone/commit/82b8654f0a3f9712188373e371bbb32e6066f8be))
* **Design Tokens:** NO-JIRA remap focus color ([e003c8b](https://github.com/dialpad/dialtone/commit/e003c8b8e38cda3a627b48eae48c1e8e6649bfbd))
* **Design Tokens:** NO-JIRA unhack tmo theme ([#669](https://github.com/dialpad/dialtone/issues/669)) ([daa0801](https://github.com/dialpad/dialtone/commit/daa0801aac82342129f1cfd13849c62063064422))
* **Design Tokens:** NO-JIRA update critical foreground and border ([ebd2885](https://github.com/dialpad/dialtone/commit/ebd288508696f4c3aa3fd720b1122d86662f6531))
* **Design Tokens:** NO-JIRA updated dark muted foreground color ([f5146cc](https://github.com/dialpad/dialtone/commit/f5146cca5eb3fbff4d3cad6faf381a6eb2510c99))
* **Design Tokens:** NO-JIRA updated purples ([099fa4b](https://github.com/dialpad/dialtone/commit/099fa4b2ed2050688142711f607558b608fe31c0))
* **Design Tokens:** NO-JIRA updated purples dark mode library connection ([a5aa5ab](https://github.com/dialpad/dialtone/commit/a5aa5ab02177c5f6c2a8cfdba4bb2e875278c7b8))
* DLT-2329 update Archivo to Season Mix font ([#648](https://github.com/dialpad/dialtone/issues/648)) ([ca8fdcf](https://github.com/dialpad/dialtone/commit/ca8fdcf6c3064d402a87be050b00fdbee92d6f88))
* DLT-2350 expand color css utility classes ([#655](https://github.com/dialpad/dialtone/issues/655)) ([1b271f3](https://github.com/dialpad/dialtone/commit/1b271f36eafbd798cdaa8c38f40202e11719909f))
* **Empty State:** DLT-2361 DLT-2362 update icon color ([#651](https://github.com/dialpad/dialtone/issues/651)) ([0be7414](https://github.com/dialpad/dialtone/commit/0be7414bd2d2ef32d2d491e638ebdd492971746c))
* **Icon:** DLT-2410 new dialpad logo icon and update gradients ([#670](https://github.com/dialpad/dialtone/issues/670)) ([c29ee5a](https://github.com/dialpad/dialtone/commit/c29ee5aa2898bab28b4494fe70050fedaff1fb86))
* **Icon:** DLT-2426 add dialpad meetings icon ([#673](https://github.com/dialpad/dialtone/issues/673)) ([96ba523](https://github.com/dialpad/dialtone/commit/96ba52320ff65ff281c4f35e744ed35c1c94fa58))
* **Icons:** [NO-JIRA] new icons dialpad-star ([#706](https://github.com/dialpad/dialtone/issues/706)) ([f38fefd](https://github.com/dialpad/dialtone/commit/f38fefd0b4bcd9de7780cb3c48b291d9de8dc29a))
* **Illustrations:** DLT-2424 add rebranded illustrations ([#693](https://github.com/dialpad/dialtone/issues/693)) ([6644761](https://github.com/dialpad/dialtone/commit/6644761f32a3854e0f404cd7416d9a12805bee38))
* **Linters:** DLT-2395 base color linter rules ([#667](https://github.com/dialpad/dialtone/issues/667)) ([a23e3c6](https://github.com/dialpad/dialtone/commit/a23e3c6b4229ecd70b1ce9766b30a8f5ba42e2ad))

francisrupert committed:
fix(dialpad-logo) NO-JIRA theme-agnostic logo (#710)

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

# Conflicts:
#	.github/workflows/visual_tests.yml
#	CHANGELOG.json
#	CHANGELOG.md
#	package.json
#	packages/dialtone-icons/CHANGELOG.json
#	packages/dialtone-icons/CHANGELOG.md
#	packages/dialtone-icons/android/gradle.properties
#	packages/dialtone-icons/package.json
#	pnpm-lock.yaml

juliodialpad committed:
update pnpm-lock

Signed-off-by: Julio Ortega <julio.ortega@dialpad.com>

braddialpad committed:
attempt to fix lock file

jcarter-dp committed:
feat(icons): DLT-2494 update dialpad logo gradient (#726)

fede-dp committed:
feat(illustrations): NO-JIRA new illustrations added (#718)

Co-authored-by: Josh Carter <joshua.carter@dialpad.com>

juliodialpad committed:
Merge branch 'staging' into rebrand-2025-beta

# Conflicts:
#	CHANGELOG.json
#	CHANGELOG.md
#	apps/dialtone-documentation/docs/_data/svg-brand-logos.json
#	package.json

francisrupert committed:
feat(icons): DLT-2494 revert update dialpad logo gradient (#727)

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.8

# [4.35.0-rebrand-2025-beta.8](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.35.0-rebrand-2025-beta.7...dialtone-icons/v4.35.0-rebrand-2025-beta.8) (2025-04-22)

### Features

* **Icons:** DLT-2494 revert update dialpad logo gradient ([#727](https://github.com/dialpad/dialtone/issues/727)) ([8b33df6](https://github.com/dialpad/dialtone/commit/8b33df695fc3160d2772b38d9a45e86704f90420))
* **Icons:** DLT-2494 update dialpad logo gradient ([#726](https://github.com/dialpad/dialtone/issues/726)) ([933bb7b](https://github.com/dialpad/dialtone/commit/933bb7b40695fa153b7d63d8533793a6208cf536))
* **Illustrations:** NO-JIRA new illustrations added ([#718](https://github.com/dialpad/dialtone/issues/718)) ([6b45b79](https://github.com/dialpad/dialtone/commit/6b45b79b2dee2e42d9c075a13b00e85f223f2763))

semantic-release-bot committed:
chore(release): NO-JIRA dialtone-icons/v4.35.0-rebrand-2025-beta.8 gradle

semantic-release-bot committed:
chore(release): NO-JIRA dialtone/v9.112.0-rebrand-2025-beta.1

# [9.112.0-rebrand-2025-beta.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.111.1...dialtone/v9.112.0-rebrand-2025-beta.1) (2025-04-22)

### Bug Fixes

* **Callbar Button With Dropdown:** DLT-2441 add modal false ([#723](https://github.com/dialpad/dialtone/issues/723)) ([b031e78](https://github.com/dialpad/dialtone/commit/b031e785b698b45b6cc85ab4a52f1b029ed1854f))
* **Contact Centers Row:** DLT-2363 update colors on recipes ([#663](https://github.com/dialpad/dialtone/issues/663)) ([aac54e6](https://github.com/dialpad/dialtone/commit/aac54e61185938ea5b73fcaae31ccfca4c5b20b4))
* NO-JIRA bring dialpad illustration assets back ([e3dc23e](https://github.com/dialpad/dialtone/commit/e3dc23eb1418b9901a6a6eabd002c2e760a7a4e3))
* NO-JIRA icons build, naming conflict and typo ([#671](https://github.com/dialpad/dialtone/issues/671)) ([de83b22](https://github.com/dialpad/dialtone/commit/de83b22354ab59fa52af91fbbdf8f8060ac5b88c))
* NO-JIRA icons-illustrations naming conflicts ([29f1f13](https://github.com/dialpad/dialtone/commit/29f1f13f1cd55af95327f7900b88d018f327ab32))
* NO-JIRA vue 3 v-model corrections ([#711](https://github.com/dialpad/dialtone/issues/711)) ([2d5c646](https://github.com/dialpad/dialtone/commit/2d5c646b268a06b1ee43b4b07646efdf353a9665))

### Code Refactoring

* DLT-2334 update design colors documentation ([#679](https://github.com/dialpad/dialtone/issues/679)) ([a22a1c0](https://github.com/dialpad/dialtone/commit/a22a1c0b138f147419229c8c182c2cd4d619e66f))

### Documentation

* **Css Utilities:** NO-JIRA update css utilities demo to emphasize semantic ([#665](https://github.com/dialpad/dialtone/issues/665)) ([520eab4](https://github.com/dialpad/dialtone/commit/520eab42a48b6d641f156cb9059ac76416dcdd24))
* DLT-2354 DLT-2331 apply season to doc site and add dialtone logo ([#652](https://github.com/dialpad/dialtone/issues/652)) ([0bd67c5](https://github.com/dialpad/dialtone/commit/0bd67c52cf8b01085b87041c329969d1f901f6ae))
* DLT-2391 color utilities documentation ([#659](https://github.com/dialpad/dialtone/issues/659)) ([4d07c12](https://github.com/dialpad/dialtone/commit/4d07c126efd6d29b54f5b0ec599127cc39e84d6c))
* DLT-2411 suppress-old-brand-icons ([#680](https://github.com/dialpad/dialtone/issues/680)) ([7f6a35e](https://github.com/dialpad/dialtone/commit/7f6a35e585694e4d594ad73d36ed2ebcf1edf568))
* modify the brand page, remove the word-mark variant, update the logo ([#668](https://github.com/dialpad/dialtone/issues/668)) ([08370f7](https://github.com/dialpad/dialtone/commit/08370f7aa45a95e1ba2e0cedb253a98215be5b2c))

### Features

* **Button:** DLT-2408 add positive kind ([#681](https://github.com/dialpad/dialtone/issues/681)) ([0265764](https://github.com/dialpad/dialtone/commit/0265764a631c8a909b1dfa288d5e21881aac0d8e))
* **Design Tokens:** DLT-2352 apply new 2025 reb colors ([#636](https://github.com/dialpad/dialtone/issues/636)) ([40d8f96](https://github.com/dialpad/dialtone/commit/40d8f9629ca770cad4a9f9cfd18002824c3302d1))
* **Design Tokens:** DLT-2458 wfm extended base colors ([#694](https://github.com/dialpad/dialtone/issues/694)) ([2905279](https://github.com/dialpad/dialtone/commit/2905279609c8abcb0ac2168328b912fc950d3dfc))
* **Design Tokens:** NO-JIRA positive button border color ([497bba8](https://github.com/dialpad/dialtone/commit/497bba84dd899e7a01dbe5a7c21741c0c4572d02))
* **Design Tokens:** NO-JIRA positive button border color (dark) ([82b8654](https://github.com/dialpad/dialtone/commit/82b8654f0a3f9712188373e371bbb32e6066f8be))
* **Design Tokens:** NO-JIRA remap focus color ([e003c8b](https://github.com/dialpad/dialtone/commit/e003c8b8e38cda3a627b48eae48c1e8e6649bfbd))
* **Design Tokens:** NO-JIRA unhack tmo theme ([#669](https://github.com/dialpad/dialtone/issues/669)) ([daa0801](https://github.com/dialpad/dialtone/commit/daa0801aac82342129f1cfd13849c62063064422))
* **Design Tokens:** NO-JIRA update critical foreground and border ([ebd2885](https://github.com/dialpad/dialtone/commit/ebd288508696f4c3aa3fd720b1122d86662f6531))
* **Design Tokens:** NO-JIRA updated dark muted foreground color ([f5146cc](https://github.com/dialpad/dialtone/commit/f5146cca5eb3fbff4d3cad6faf381a6eb2510c99))
* **Design Tokens:** NO-JIRA updated purples ([099fa4b](https://github.com/dialpad/dialtone/commit/099fa4b2ed2050688142711f607558b608fe31c0))
* **Design Tokens:** NO-JIRA updated purples dark mode library connection ([a5aa5ab](https://github.com/dialpad/dialtone/commit/a5aa5ab02177c5f6c2a8cfdba4bb2e875278c7b8))
* DLT-2329 update Archivo to Season Mix font ([#648](https://github.com/dialpad/dialtone/issues/648)) ([ca8fdcf](https://github.com/dialpad/dialtone/commit/ca8fdcf6c3064d402a87be050b00fdbee92d6f88))
* DLT-2350 expand color css utility classes ([#655](https://github.com/dialpad/dialtone/issues/655)) ([1b271f3](https://github.com/dialpad/dialtone/commit/1b271f36eafbd798cdaa8c38f40202e11719909f))
* **Empty State:** DLT-2361 DLT-2362 update icon color ([#651](https://github.com/dialpad/dialtone/issues/651)) ([0be7414](https://github.com/dialpad/dialtone/commit/0be7414bd2d2ef32d2d491e638ebdd492971746c))
* **Icon:** DLT-2410 new dialpad logo icon and update gradients ([#670](https://github.com/dialpad/dialtone/issues/670)) ([c29ee5a](https://github.com/dialpad/dialtone/commit/c29ee5aa2898bab28b4494fe70050fedaff1fb86))
* **Icon:** DLT-2426 add dialpad meetings icon ([#673](https://github.com/dialpad/dialtone/issues/673)) ([96ba523](https://github.com/dialpad/dialtone/commit/96ba52320ff65ff281c4f35e744ed35c1c94fa58))
* **Icons:** [NO-JIRA] new icons dialpad-star ([#706](https://github.com/dialpad/dialtone/issues/706)) ([f38fefd](https://github.com/dialpad/dialtone/commit/f38fefd0b4bcd9de7780cb3c48b291d9de8dc29a))
* **Icons:** DLT-2494 revert update dialpad logo gradient ([#727](https://github.com/dialpad/dialtone/issues/727)) ([8b33df6](https://github.com/dialpad/dialtone/commit/8b33df695fc3160d2772b38d9a45e86704f90420))
* **Icons:** DLT-2494 update dialpad logo gradient ([#726](https://github.com/dialpad/dialtone/issues/726)) ([933bb7b](https://github.com/dialpad/dialtone/commit/933bb7b40695fa153b7d63d8533793a6208cf536))
* **Illustrations:** DLT-2424 add rebranded illustrations ([#693](https://github.com/dialpad/dialtone/issues/693)) ([6644761](https://github.com/dialpad/dialtone/commit/6644761f32a3854e0f404cd7416d9a12805bee38))
* **Illustrations:** NO-JIRA new illustrations added ([#718](https://github.com/dialpad/dialtone/issues/718)) ([6b45b79](https://github.com/dialpad/dialtone/commit/6b45b79b2dee2e42d9c075a13b00e85f223f2763))
* **Linters:** DLT-2395 base color linter rules ([#667](https://github.com/dialpad/dialtone/issues/667)) ([a23e3c6](https://github.com/dialpad/dialtone/commit/a23e3c6b4229ecd70b1ce9766b30a8f5ba42e2ad))

jcarter-dp committed:
feat(icons): DLT 2494 update dialpad logo gradient (#729)

jcarter-dp committed:
feat(illustrations): DLT-2489 add rebrand spot illustrations (#730)

braddialpad committed:
Merge branch 'staging' into rebrand-2025-beta
